docs/history.md

```diff
@@ -1,5 +1,27 @@
 # History
 
+## 3.67.0 (2022-10-13)
+
+Critical Changes
+
+-   The `get_github_api` method signature changed from `get_github_api(repo: str = None, owner: str = None)` to `get_github_api(url:str = None)`(#3256)
+-   CumulusCI will no longer be supporting installations via `brew`. See our docs for the officially supported install method via pipx.(#3382)
+-
+
+Changes
+
+-   When using playwright instead of Selenium, we now have limited support writing keywords in Javascript. For more information see the topic "Writing keywords in javascript" in the documentation under "Playwright Technology Preview"(#3378)
```

Comment:
```suggestion
-   When using Playwright instead of Selenium, we now have limited support writing keywords in JavasSript. For more information see the topic "Writing keywords in JavaScript" in the documentation under "Playwright Technology Preview". (#3378)
```